pkgs/by-name/gi/git-autoshare/package.nix

```
@@ -0,0 +1,28 @@
+{ fetchFromGitHub
+, lib
+, python3Packages
+}:
+python3Packages.buildPythonApplication rec {
+  pname = "git-autoshare";
+  version = "1.0.0b6";
+  pyproject = true;
```

```suggestion
  pyproject = false;
```
it doesn't seem there's a pyproject.toml in the root directory

---

Quoting from https://nixos.org/manual/nixpkgs/stable/#buildpythonpackage-parameters

> you can use pyproject = true even if the package only has a setup.py

---

huh, TIL. thanks!